7.0.0/oracle/shared_db_cleanup.sh

```
@@ -0,0 +1,126 @@
+      FAILED_LOG="$LOG_DIR/failed_tenant_${TENANT_ID}.log"
+      echo "$DELETE_PROCEDURE" > "$FAILED_LOG"
+      echo "Failed to process tenant ID $TENANT_ID with org ID $ORG_UUID. Check log: $FAILED_LOG" | tee -a "$SUMMARY_FILE"
+    fi
+
+    echo "Completed deletion for TENANT_ID=$TENANT_ID."
+  else
+    echo "Skipping invalid or empty line: TENANT_ID=$TENANT_ID, ORG_UUID=$ORG_UUID"
+  fi
+done < "$EXPORT_FILE"
+
+echo "Data deletion process completed. Summary available at $SUMMARY_FILE."
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)